src/apps/packet_filter/README.md.src

```diff
@@ -54,15 +74,11 @@ is allowed.
 
 — Key **state_track**
 
-*Optional*. Tracks connections state in a named connection table.  If used
-inside a rule, every packet that passes this specific rule is tracked.
-If used 'outside' any rule, packets passing _any_ rule on this app will be
-tracked.
+*Optional*. A string naming a connection table. If set, packets passing
+the rule are tracked in the specified connection table.
 
 — Key **state_check**
 
-*Optional*. Checks if a packet belongs to a connection in a named
-connection table.  If used inside a rule, a packet must belong to an
-existing connection in addition to any other condition in the rule to pass.
-If used 'outside' any rule, if a packet belongs to an existing connection
-it's enough to pass the app.
+*Optional*. A string denoting a connection table. If set, packets must
```

Comment:
Inconsistent with the **state_check** description above?

I expect the logic to be that a packet must match the rules *OR* match a known connection, but not both. The connection table is essentially a set of exceptions to the rules that allow all packets belonging to connections that we allowed to be established. (In other words, if we allow the first packet for a connection then we will allow all the future packets too, even if they don't match the rules e.g. because they are flowing in the opposite direction.)